### DISCUSSION TITLE
The result after trying it is not ideal, it can't be used at all

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win#25] [Q&A]
The machine is running Win7 X64 Computer City Enterprise Lite ghost system, the CPU is Core 2 Duo i3 2120, and the antivirus software is Kaspersky 21.8 and Adguard. At first, the wrong AVX2 installation package could not be installed. After switching the AVX package, the installation was completed normally. Now, the situation is that the task page cannot be opened, but everything is normal when I switch to the Firefox browser. I built the Kaspersky trust program, and then uninstalled Adguard and Kaspersky, and tested it; the problem remains, and the Firefox browser is completely unaffected. Influence, why? I hope that all the development and testing masters will analyze the crux of the cause of the positioning problem; I am very grateful! .

**Comment by gz83** [comment]
> The machine is running Win7 X64 Computer City Enterprise Lite ghost system, the CPU is Core 2 Duo i3 2120, and the antivirus software is Kaspersky 21.8 and Adguard. At first, the wrong AVX2 installation package could not be installed. After switching the AVX package, the installation was completed normally. Now, the situation is that the task page cannot be opened, but everything is normal when I switch to the Firefox browser. I built the Kaspersky trust program, and then uninstalled Adguard and Kaspersky, and tested it; the problem remains, and the Firefox browser is completely unaffected. Influence, why? I hope that all the development and testing masters will analyze the crux of the cause of the positioning problem; I am very grateful! .

@whxuezhe Hello, what exactly is the page you are talking about that cannot be opened? Please specify clearly.

**Reply by whxuezhe** [reply]
@gz83 https://www.265.com Most of them are https URLs, and a few are http; almost all of my commonly used URLs are included, but they cannot be accessed normally and display the content of the webpage.

**Reply by gz83** [reply]
> @gz83 https://www.265.com<img alt="" width="16" height="16" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAKfSURBVHgBjVNLTBNRFL13PrYlUMpHmKYfKrb+2KAx0bihJrpSDIkbw0YiLogbxLhyx9ZoXBnjSo0bFkRi3LAyGBMTiCJx1UCIkk5lJCLMTFtoOzPX915LQSOJdzKf9+5555533xkZ9ol4PDraGm2Z7jzSPtIADaZl2Qv/wsl/T0SjWrot3j7VdSN2vffBscDhYS2EQAO+Ld9QAPwrlpXP7MXjzoemaQlVlZ+FB7vTiZFu8DUjSFKFFMkFGR0sfS/S0mMds1PrzysVd9wwjG91BVxu8LQ2nXx4rqujP47olwhBXEhULaI2yRi+0AyJq629pdXyqPzrAJim/U7mkkOXUhOJ+32gtgWwKgvxw6k32HElDkqTyka4o5fURhmjl1twK1fq83R6LfF5RQuCSyo5pBB7g+spAu+ATC4oVNArsD5ng+ep6AHLkwz+iJ/x+kKCgEgCx1OQL3Q9uU7gsblCtoKfhz+CmSkSw3Acy6tijSgugGzgkQKMHl2J5WrN5USLd2ehNR2G8GASPXIYmCUYhq0RmF0FbAs1+cgr8MhnLCgubkLHYIrlFKpWFzdSzQFSTQF5u0mhhocvdRDarx2FpTuzWDYJGQnwm/fKI9wlIH7SXjXh7GkiJ+q8eZIpRMg+zYi8KOAqbLcSiR4QyZuOYQpZtU1xbky9v8WM5LCBA8mX/cTN5LkO6xGK49zWSwIu27ZtBNa20VleS6vHo4CNfqz7k/mo5lZuDfEqLm3A8r05/PnWGMvlchOiE5ZlzQTzWy/yk59aHMPqVZNhkht9WLc6e7p2GX48mYeV8dkZd80+n82uTv/xL+xELBYbUCLBRw0XexKhoTOAhQLYk/Ngvvqy4GyWx3Rdn4H/iVgsMnTobM/XrhPdG5FI5PZ+uN9/hz4cOxubdAAAAABJRU5ErkJggg=="> Most of them are https URLs, and a few are http; almost all of my commonly used URLs are included, but they cannot be accessed normally and display the content of the webpage.

@whxuezhe No abnormality was found in the local test, and the website you mentioned can be opened normally.

Please make sure to use the system installed on the system image downloaded through Microsoft official channels or other authoritative channels for relevant tests, and please close the software or extensions that may affect the web browser.
